IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -20,7 +20,7 @@ public class ReachFive: NSObject {
     internal var scope: [String] = []
     internal let storage: Storage
     let codeResponseType = "code"
-
+    
```

Comment:
The formatter indents empty lines?

Reply:
Yes.
The style was very inconsistent, with a lot of code with these indented empty lines, and a lot of code without.
But it seems that indented empty lines was more frequent and older, so I decided to make it the default.
I could've gone the opposite way, but in the end I don't really care, just that the style is consistent.